osquery/tables/system/darwin/extended_attributes.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
```

Comment:
There is no control character, I edited that line manually after pasting it in because the output came from an older version.